### PR TITLE
AWS VPC peering article

### PR DESCRIPTION
**Pull request by dewan-ahmed**
# What changed, and why it matters

This is a migration task which resolves https://github.com/aiven/devportal/issues/1106

Original help article: https://help.aiven.io/en/articles/778836 (broken down into multiple articles)

### CODE REVIEW

**Review by encima — APPROVED** [review]

File: docs/platform/howto/manage-vpc-peering.rst [hunk 1]
```diff
@@ -34,6 +34,7 @@ Cloud-specific VPC peering instructions
 -----------------------------------------------------
 
 - :ref:`Set up VPC peering on Google Cloud Platform (GCP) <vpc-peering-gcp>`.
+- :ref:`Set up VPC peering on Amazon Web Services (AWS) <vpc-peering-aws>`.
```
Comment (ftisiot) [inline docs/platform/howto/manage-vpc-peering.rst:37]: Should this be a `:doc:` and not a `:ref:` ?

File: docs/platform/howto/vpc-peering-aws.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+.. _vpc-peering-aws:
```
Comment (ftisiot) [inline docs/platform/howto/vpc-peering-aws.rst:1]: I believe once we move the above reference to a doc, this becomes useless

**Review by ftisiot — DISMISSED** [review]
A couple of modifications needed for consistency

**Comment by dewan-ahmed** [comment]
Hi @ftisiot I think I addressed the review comments already but GitHub is still showing "1 change requested". Pls let me know in case I missed anything. Thanks.

**Pull request merged**